app/views/passengers/show.html.erb

```diff
@@ -0,0 +1,27 @@
+  <div class = "container">
+  <% if @passenger.nil? %>
+    <h1> 404 Not found</h1>
+  <% else %>
+    <h1><%=@passenger.name %></h1>
+    <button>
+      <%= link_to "Add A Trip", new_trip_path %>
```

Comment:
You should make this route be for the `passenger_trip_path`  so the passenger can be pre-set.